gapil/template/constant_sets.go

```diff
@@ -188,6 +188,25 @@ func buildConstantSets(api *semantic.API, mappings *resolver.Mappings) *Constant
 			constsets[p] = b.addLabels(ev.Labels, ev.Ty.IsBitfield)
 		}
 	}
+	for _, e := range api.Enums {
+		unique := false
+		for _, annotation := range e.Annotations {
+			if annotation.Named == semantic.Named("unique") {
+				unique = true
+				break
+			}
+		}
+		if unique {
```

Comment:
You can simplify this with
if unique := e.Annotations.GetAnnotation("unique"); unique != nil {